epub33/a11y/index.html

```diff
@@ -915,83 +925,96 @@ <h6>Skippability</h6>
 										colours or placement so readers can filter this information visually out while
 										reading).</p>
 									<p>Readers who prefer auditory playback, however, cannot skip this information with
-										the same ease by default. Media overlays playback will typically result in
-										Reading Systems rendering secondary content where it occurs.</p>
-									<p>When EPUB Creators add structural semantics to Media Overlay Documents, however,
-										Reading Systems can create reading experiences that allow users to decide which
-										secondary content to skip by default during playback.</p>
+										the same ease by default. Synchronized text-audio playback will typically result
+										in Reading Systems rendering secondary content where it occurs.</p>
+									<p>When EPUB Creators add structural semantics, however, Reading Systems can create
+										reading experiences that allow users to decide which secondary content to skip
+										by default during playback.</p>
 								</dd>
 
-								<dt id="sec-mo-skippability-conf">Meeting this Objective</dt>
+								<dt id="sec-sync-skippability-conf">Meeting this Objective</dt>
 								<dd>
-									<p>EPUB Creators SHOULD identify all <a
-											href="https://www.w3.org/TR/epub/#sec-skippability">skippable structures</a>
-										[[EPUB-3]] in Media Overlay Documents.</p>
+									<p>EPUB Creators SHOULD identify all skippable structures if the synchronized
+										text-audio playback format does not handle escaping content by default.</p>
+									<div class="note">
+										<p>Refer to <a data-cite="epub-a11y-tech-11#sync-003">Identifying skippable
+												structures</a> [[EPUB-A11Y-TECH-11]] for more information on meeting
+											this objective.</p>
+									</div>
 								</dd>
 							</dl>
 						</section>
 
-						<section id="sec-mo-escapability">
+						<section id="sec-sync-escapability">
 							<h6>Escapability</h6>
 
 							<dl>
-								<dt id="sec-mo-escapability-obj">Objective</dt>
+								<dt id="sec-sync-escapability-obj">Objective</dt>
 								<dd>
 									<p>Enable users to automatically escape from structured content.</p>
 								</dd>
 
-								<dt id="sec-mo-escapability-understand">Understanding this Objective</dt>
+								<dt id="sec-sync-escapability-understand">Understanding this Objective</dt>
 								<dd>
 									<p>When reading visually, users can quickly move through, and escape from, highly
 										structured content such as tables, lists, and figures. Table layouts that use
```

Comment:
Suggest change to "sidebars, lists, and figures." And also remove the parts about table layout. I know we've always used tables as an escapability example but I haven't seen any good implementations and I think it's because sync text/audio doesn't really help you here. For example, you can't have a row-based presentation and switch to a column-based presentation. And even to enable basic table reading features like escaping along the axis of cell=>row=>table or scanning the column headers requires really thorough parallel content doc <-> sync text format semantic markup plus a clever reading system.

Reply:
> And also remove the parts about table layout.

Is this only for the description here or are you suggesting we should not recommend identifying tables in the techniques, too?

I don't know how well escaping works from tables, but if we don't recommend identifying them do we expect users will drop out of MO playback mode to traverse/escape tables?

If so, should we say that in a note in the techniques to explain why it's not there?

Should we also remove the recommendation from the media overlays section on escapability, too?

Does this work for everyone? @avneeshsingh @GeorgeKerscher @clapierre @gregoriopellegrino

Reply:
Okay, so recommend to identify `table` but not `table-row` and `table cell`?

Does the same apply to a list? Should we only recommend escaping from lists and not list items?

Reply:
We can still "encourage" the substructures be identified without recommending, too.

Reply:
I've added the following note in the latest commit:

> Sometimes escapable structures may contain escapable structures. For example, tables are composed of many rows and cells that users may want to separately escape from. While identifying the nested escapable structures is encouraged, it is not required at this time as the process of escaping from nested structures is not well defined.

Reply:
> Okay, so recommend to identify `table` but not `table-row` and `table cell`?
> 
To me, it comes down to author intent. As in, what is the structure they would like to offer the user the option to escape from?

Reply:
> While identifying the nested escapable structures is encouraged, it is not required at this time as the process of escaping from nested structures is not well defined.

I'm afraid that identifying all the nested structures would break the most useful behavior provided by a simple implementation. Eg am RS that uses the rule "if semantic exists, offer escape" and therefore let's you escape cells but isn't clever enough to expose any wrapping escapable containers.

Reply:
But if it's purely author intent, then doesn't that make an accessibility requirement/recommendation virtually impossible? You can't verify what someone thinks makes for the best escaping.

And if we try to make people choose only one level when there is nesting (e.g., only identify cells or the table),  it also feels like we're pushing people to only accommodate the least useful escaping possible.

Reply:
What if we remove the note and just start with a recommendation to identify the larger structures.

These are only optional requirements, after all, so if we find the guidance is impractical we can always tweak the suggestions later before they become recommendations to follow.

Reply:
The last commit makes a number of changes:

I've taken the note that I put in earlier and moved it to the core specification. I've also modified it to make clear that reading system support is iffy:

> Sometimes escapable structures may contain escapable structures. For example, tables are composed of many rows and cells that users may want to separately escape from. Reading System support for escaping from such structures is complex and not well supported at this time. EPUB Creators should avoid identifying nested escapable structures until better support is available.

I've replaced the note in the techniques document with a shorter explanation that references across to the core spec section.

I've removed table-cell, table-row and list-item from the core spec's list of recommended escapable structures (and added links to the SSV for all the terms we were citing). Alternatively, I added `aside` as a recommended term for identifying sidebars.

Hopefully that'll get us over the finish line! 😄